app/modules/booking/service.server.ts

```diff
@@ -297,7 +297,7 @@ export async function upsertBooking(
       //update
       const res = await db.booking
         .update({
-          where: { id },
+          where: { id, organizationId },
```

Comment:
Shouldnt we make organizationId required and always use it? I dont understand why we use it only in this one case for `upsertBooking`